### PR TITLE
Complete jQuery AJAX diagnostic

### PR DESCRIPTION
**Pull request by xiangcatherine**
Feel comfortable with curl requests, but unsure about AJAX requests.

Comfort: 3.5
Clarity: 4

### CODE REVIEW

**Comment by xiangcatherine** [comment]
Just realized that for the AJAX requests, I could've referenced the api.js files from the examples we did in class. So I definitely know what I did wrong for the AJAX requests. 

Thank you!!

File: diagnostic.md [hunk 1]
```diff
@@ -21,13 +21,14 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+#!/bin/bash
+curl --include --request GET http://www.example.com
```
Comment (benjimelito) [inline diagnostic.md:25]: want to include a `/donuts` on the end here

File: diagnostic.md [hunk 2]
```diff
@@ -36,13 +37,14 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
+#!/bin/bash
+curl --include --request GET http://www.example.com/donuts/5
```
Comment (benjimelito) [inline diagnostic.md:41]: 👍

**Comment by benjimelito** [comment]
Looks like you were able to get some of the later jQuery questions. The earlier ones follow the same structure. 7/10

**Pull request closed without merging**